picasso-pollexor/src/test/java/com/squareup/picasso/pollexor/PollexorRequestTransformerTest.java

```diff
@@ -1,19 +1,24 @@
 package com.squareup.picasso.pollexor;
 
 import android.net.Uri;
-import com.squareup.picasso.Request;
+import com.squareup.picasso.*;
```

Comment:
Guh. It's so weird when this happens because i usually turn of wildcard imports too. I think applying the square style on this project maybe reset the imports count -_-